### PR TITLE
fix: preview modal ui to match figma

### PR DESCRIPTION
**Pull request by ilasw**
## Changes

- Source avatar too big
- Close button not aligned to right
- Too much padding in modal body (32 vs 24)
- Shared post has too much margin
- Content being rendered even if not present (with margins)

| Before | After |
|--------|--------|
| image | image | 

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://fix-modal-preview-ui.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx [hunk 1]
```diff
@@ -67,7 +68,7 @@ function PostModerationModal({
       size={Modal.Size.Large}
       kind={Modal.Kind.FlexibleTop}
     >
-      <Modal.Body className="gap-6">
+      <Modal.Body className="gap-6 !p-6">
```
Comment (AmarTrebinjac) [inline packages/shared/src/components/modals/squads/PostModerationModal.tsx:71]: Can we not override the padding without using important? I think we should avoid it whenever necessary

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx [hunk 2]
```diff
@@ -85,16 +86,14 @@ function PostModerationModal({
...
-              <SquadModerationItemContextMenu
-                id={id}
-                onDelete={onDeleteClick}
-              />
-            </div>
+            <SquadModerationItemContextMenu id={id} onDelete={onDeleteClick} />
           )}
-          <ModalClose onClick={modalProps.onRequestClose} />
+          <ModalClose
+            className="relative !right-0"
```
Comment (AmarTrebinjac) [inline packages/shared/src/components/modals/squads/PostModerationModal.tsx:92]: Same here

**Review by AmarTrebinjac — CHANGES_REQUESTED** [review]
Looks good!

Some comments about using "Important"

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx (hunk 1 above)
Comment (ilasw) [reply packages/shared/src/components/modals/squads/PostModerationModal.tsx:71]: Unfortunately this is not possible due to the order in which the classes are written by TailwindCSS. 
We have discussed this issue internally in the past and proposed using [twMerge](https://www.npmjs.com/package/tailwind-merge) but it has not been adopted yet :(

**Review by sshanzel — APPROVED** [review]

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/components/modals/squads/PostModerationModal.tsx:92]: ModalClose actually accepts props to adjust right- :)

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/modals/squads/PostModerationModal.tsx:71]: For this it's fine, ideally all modals follow guidelines, but this is a weird modal I guess.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Please use props on the close one

File: packages/shared/src/components/modals/squads/PostModerationModal.tsx (hunk 2 above)
Comment (ilasw) [reply packages/shared/src/components/modals/squads/PostModerationModal.tsx:92]: Nice! Didn't see before, thanks for suggesting! 🙏

**Pull request merged**